### PR TITLE
Improve build times by disabling wasm-builder in `no_std`

### PR DESCRIPTION
**Pull request by bkchr**
There is no need to compile wasm-builder in `no_std` as we skip the compilation of the crate any way. So, we can also directly disable wasm-builder as dependency in `no_std` to improve build times.

### CODE REVIEW

**Comment by xlc** [comment]
parachain teams should do similar change to improve build time?

**Comment by bkchr** [comment]
> parachain teams should do similar change to improve build time?

You don't need to, but you can.

File: parachains/runtimes/testing/rococo-parachain/build.rs [hunk 1]
```diff
@@ -1,25 +1,26 @@
-// Copyright 2019-2021 Parity Technologies (UK) Ltd.
-// This file is part of Cumulus.
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
```
Comment (mrcnski) [inline parachains/runtimes/testing/rococo-parachain/build.rs:2]: We're not putting the year anymore?

I think all the licenses should be updated in a dedicated PR, as this distracts from other changes here and is incomplete in itself. We have a [license scanner](https://github.com/paritytech/license-scanner) tool for this which we already used in Polkadot. cc @rzadp

Comment (bkchr) [reply parachains/runtimes/testing/rococo-parachain/build.rs:2]: Not really sure that it distracts from the changes. Yes we are not putting the year anymore.

**Review by koute — APPROVED** [review]

**Review by skunert — APPROVED** [review]

File: parachains/runtimes/testing/rococo-parachain/build.rs (hunk 1 above)
Comment (rzadp) [reply parachains/runtimes/testing/rococo-parachain/build.rs:2]: Hey, feel free to ask me to set up the licence scanner CI check here if it has proven useful in Polkadot.
